code/modules/research/experimentor.dm

```diff
@@ -241,6 +241,7 @@
 	smoke.start()
 
 
+/* yogs start - moved to yogstation/ folder

```

Comment:
You don't have to comment this out

Reply:
yeah but isn't it more clear if it is? like someone oculd look at the code and be like "WTF the code says it does this but it doesn't" without realizing that it got mirrored somewhere else?